scripts/download-core.sh

```diff
@@ -4,7 +4,7 @@ set -e
 set -o pipefail
 
 # Set to "latest" for the latest build.
-: ${REALM_CORE_VERSION:=1.1.2}
+: ${REALM_CORE_VERSION:=1.4.0}
```

Comment:
Do we need to change the downloaded tarball filename format as well? In the object-store changes we now seem to be using `xz` extension rather than `bz2`. Also we may need to replace the `gz` extensions for Darwin and Node with `xz`?